.github/workflows/ci-cd.yml

```
@@ -159,7 +153,74 @@ jobs:
       - if: steps.publish.outputs.released == 'true'
         uses: actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a # v7
         with:
-          name: release-info
-          path: release-info.txt
-          if-no-files-found: error
-          retention-days: 2
+          ref: ${{ github.event_name == 'schedule' && github.sha || needs.release.outputs.tag }}
```

actionlint (1.7.12)

[error] property "release" is not defined in object type {build: {outputs: {released: string; tag: string; version: string}; result: string}}

(expression)